packages/gasket-fetch/browser.js

```diff
@@ -0,0 +1,9 @@
+
+exports = window.fetch; // To import fetch from @gasket/fetch
+exports.default = window.fetch; // For TypeScript consumers without esModuleInterop.
+exports.fetch = window.fetch; // To import {fetch} from @gasket/fetch
```

Comment:
I don't think we need this. The API should for the browser module should be the same as for node, which does not support `{ fetch }` import and I don't think there's a case for it.